adobe/serializable.hpp

```diff
@@ -110,7 +113,7 @@ class serializable_t {
 
     serializable_t(serializable_t&& x) : instance_m(x.instance_m.release()) {}
 
-    void operator()(std::ostream& s) const { object()(s); }
+    void operator()(std::ostream& s) const { object()._out(s); }
```

Comment:
If you're gonna change stuff you should give it a doc comment along the way; just sayin'.

Reply:
Good point - I'll try and do better.